timezone_converter/main.py

```diff
@@ -71,6 +72,13 @@ def build_parser() -> argparse.ArgumentParser:
         dest='hour',
         help='show a single hour',
     )
+    parser.add_argument(
+        '--search',
+        nargs='?',
+        metavar='STRING',
```

Comment:
Here I would change `STRING` to `WORD`

Reply:
I also forgot to mention the addition of `-S` as a short version that we talked about in the issue...